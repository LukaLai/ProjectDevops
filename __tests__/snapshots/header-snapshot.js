import { render } from '@testing-library/react'
import Header from '../../src/app/_components/header'

it('renders homepage unchanged', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot()
})