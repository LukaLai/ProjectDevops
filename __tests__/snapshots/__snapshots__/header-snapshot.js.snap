// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <h2
    class="text-2xl md:text-4xl font-bold tracking-tight md:tracking-tighter leading-tight mb-20 mt-8"
  >
    <a
      class="hover:underline"
      href="/"
    >
      Blog
    </a>
    .
  </h2>
</div>
`;
